hack/install-jq.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPTS_DIR="$(dirname "$0")"
+# Provides $OS,$ARCH,$PLATFORM,$ROOT variables
+source "$SCRIPTS_DIR/install-common.sh"
+
+cleanup() {
+  rm -rf "$WORK_DIR"
+}
+trap "cleanup" EXIT SIGINT
```

Comment:
Is this still needed as it is already done inside `install-common.sh` ?
https://github.com/DataDog/watermarkpodautoscaler/blob/02ebc32ba5e2eaa8600c77f154c915efd0652176/hack/install-common.sh#L46-L52

Reply:
no,
I copy pasted the install-yq.sh script

I will do the change in both